semanticsearch/README.md

```diff
@@ -0,0 +1,440 @@
+# Semantic Search Workload
+
+This workload is to benchmark performance of search of Semantic Search queries of OpenSearch. 
+
+## Datasets
+
+This workload is based on a [daily weather measurement from NOAA](ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/).
+
+To recreate the document corpus:
+
+1. Download the following files:
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/2014.csv.gz
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/2015.csv.gz
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/2016.csv.gz
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/ghcnd-stations.txt
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/ghcnd-countries.txt
+    * ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/ghcnd-states.txt
+2. Decompress measurement files. For example: `gunzip 2016.csv.gz`
+3. Sort the files by station. For example: `sort --field-separator=',' --key=1,2 -o 2016-sorted.csv 2016.csv`
+4. Execute a script like `_tools/process.py` to create json documents.
```

Comment:
Please elaborate on this and provide references for where to obtain this script for users who aren't familiar with `_tools/process.py`

Reply:
As I'm going to remove this section this is not going to be relevant